stages/org.osbuild.rpm

```diff
@@ -111,6 +111,10 @@ SCHEMA_2 = """
       "description": "Prevent dracut from running",
       "type": "boolean"
     },
+    "first_boot": {
```

Comment:
Pardon my ignorance  - I was wondering why this option (or the `"machine_id": ["no","empty","preserved"]`) is part of the rpm stage (and would we potentially need it for pacman as well)?  Is there a deeper reason or is it just expedient this way (e.g. because a stage like `org.osbuild.machine-id` that that would be silly)?

Reply:
What we do works in 99% cases. https://systemd.io/BUILDING_IMAGES/ is a good source on how to make it better. I tried some experiments with definining `machine-id` as `uninitialized`, but I think that `kernel-install` complained. `/etc/kernel/entry-token` might solve the issue, but it isn't supported on RHEL 8. So I just gave up. It might be worth revisiting this topic in Fedora. An advantage is that `ConditionFirstBoot=` currently doesn't work on Fedora nor RHEL and fixing `machine-id` would fix this as well.

Reply:
Thanks for your inputs here. I read over the man page the the BUILDING_IMAGES url and it seems to me we could try to move this into it's own stage, I played around with this in https://github.com/osbuild/osbuild/compare/main...mvo5:stages/machine-id?expand=1

The commit messages describe the idea and there is a stage test that uses systemd-nspawn to validate things work in the way we expect them. The tl;dr is that since systemd v247 the "unitiialized" value can be used (thanks Ondrej for pointing this out!). 

Please let me know if this looks sensible - in this case I will open the new PR and close(?) this one (referencing to the new PR of course).